build.sbt

```diff
@@ -19,6 +19,16 @@ resolvers ++= Seq[Resolver](
 
 mainClass in (Compile, run) := Some("uk.gov.nationalarchives.tdr.api.http.ApiServer")
 
+val compareSchema = taskKey[Unit]("Generate the graphql schema and compares it with schema.graphql in the project root")
+
+compareSchema := {
```

Comment:
Just wondering if you saw the `graphqlValidateSchema` task in sbt-graphql, and whether it's worth using it since it looks like it might compare the schemas logically rather than byte-for-byte? https://github.com/muuki88/sbt-graphql#schema-comparison

Reply:
Ah good spot yeah. I've added the schema from the file to build sbt and then you can compare them using the `graphqlValidateSchema` It's better because it will tell you which fields have been added or removed rather than just telling you they're different.